package.json

```diff
@@ -11,15 +11,19 @@
   },
   "dependencies": {
     "@4chain-ag/react-configuration": "^1.0.4",
+    "@bsv/sdk": "^1.0.37",
     "@bsv/spv-wallet-js-client": "file:../spv-wallet-js-client",
```

Comment:
This doesn't work for me. 
Please use some actual version of the package. 

Of course, you can do it locally 😄

Reply:
Changed to actual version.

Reply:
I think, we should wait for https://github.com/bitcoin-sv/spv-wallet-js-client/pull/997. 
Because, without it, you cannot call anything via `js-client` 🤔

Reply:
Merged PR above